SECURITY.md

```diff
@@ -0,0 +1,152 @@
+### Operating an XRP Ledger server securely
+
+For more details on operating an XRP Ledger server securely, please visit https://xrpl.org/manage-the-rippled-server.html.
+
+
+# Security Policy
+
+## Supported Versions
+
+Software constantly evolves. In order to focus resources, we only generally only accept vulnerability reports that affect recent and current versions of the software. We always accept reports for issues present in the **master**, **release** or **develop** branches, and with proposed, [open pull requests](https://github.com/ripple/rippled/pulls).
```

Comment:
Consider removing "we only accept vulnerability reports for recent." An older version can still have security-related issues. Per Ed, maybe we can do something like "we will only consider vulnerability reports for old version if the vulnerability still exists in the recent and current versions.". The idea here is not to discourage people from submitting issues.

Reply:
Soliciting the reporting of vulnerabilities that are not present in a recent version of the software (i.e. are no longer present in the source code) will only serve to bottleneck the triage and evaluation process, with little to no actual benefit.

By way of example, assume a bug against **1.2.4** is reported _today_ which was fixed in **1.4.0**. The **1.5.0** release is out and **1.6.0** is in beta. The only actionable item from a report would be "please upgrade to **1.5.0** which does not contain this bug."